cli/src/main/resources/META-INF/native-image/org.eclipse.hono/hono-cli/native-image.properties

```diff
@@ -0,0 +1,12 @@
+# Copyright (c) 2023 Contributors to the Eclipse Foundation
+#
+# See the NOTICE file(s) distributed with this work for additional
+# information regarding copyright ownership.
+#
+# This program and the accompanying materials are made available under the
+# terms of the Eclipse Public License 2.0 which is available at
+# http://www.eclipse.org/legal/epl-2.0
+#
+# SPDX-License-Identifier: EPL-2.0
+
+Args = -H:+RunReachabilityHandlersConcurrently
```

Comment:
this file is not needed as the cli does not depend on `gax-rpc` ....

Reply:
Removed.